Casks/b/bluej.rb

```diff
@@ -8,15 +8,7 @@
   homepage "https://www.bluej.org/"
 
   livecheck do
-    url "https://www.bluej.org"
-    regex(%r{href=.*?/BlueJ-mac-(\d+)(\d+)(\d+)(a)?\.dmg}i)
-    strategy :page_match do |page|
-      match = page.match(regex)
-      next if match.blank?
-
-      "#{match[1]}.#{match[2]}.#{match[3]}" unless match[4]
-      "#{match[1]}.#{match[2]}.#{match[3]}#{match[4]}"
-    end
+    url "https://github.com/k-pet-group/BlueJ-Greenfoot"
```

Comment:
```suggestion
    url :homepage
    regex(/Version\s*v?(\d+(?:\.\d+)+)/i)
```

As mentioned in a DM, the homepage provides version information and links to the cask `url`, so we should continue checking that (i.e., the cask `url` isn't a Git tag or release asset). The page contains text like "Version 5.3.0", so we can match that instead of naively inserting dots into the dotless filename version.

The looseness of this may eventually cause problems but it's fine for now. If we run into problems down the road that we can't fix through a regex tweak, we can switch to the versions page (https://www.bluej.org/versions.html) and use a stricter regex (e.g., `/>\s*v?(\d+(?:\.\d+)+)\s*</i`). That page is almost 7x larger (gzipped), so checking the homepage like this is fine for now.

Reply:
Thank you @samford! 🙏